common.go

```diff
@@ -133,9 +133,6 @@ func ConvertToBytes(humanReadableString string) int {
 		}
 	}
 
-	if lastNumberPos < 0 {
```

Comment:
This condition never happens because line 124 has no initialization.

Reply:
Looks like `lastNumberPos` will be 0 if no digits are encountered during the loop, as negative values are precluded by the loop conditions (`for i := strLen - 1; i >= 0; i--`). Therefore, lastNumberPos cannot be less than 0.